components/ui/button.tsx

```diff
@@ -3,27 +3,31 @@ import { Slot } from "@radix-ui/react-slot";
 import { cva, type VariantProps } from "class-variance-authority";
 
 import { cn } from "@/lib/utils";
+import { ReloadIcon } from "@radix-ui/react-icons";
```

Comment:
we are already using `lucide-react`, use this instead `import { Loader2 } from "lucide-react"`
Remove this library